### ISSUE TITLE
"Filmic" tone mapping is not properly ported

### ISSUE THREAD

**Issue by rhys-vdw** [AdamFrisby/Post-Stack-Converter#2]
The color grading "mode" needs to be set to "High Definition Range" for this option to be available.

**Comment by AdamFrisby** [comment]
That breaks some other options, I think the YRGB curves only work in LDR.

On 13 Dec. 2018 7:15 pm, Rhys van der Waerden <notifications@github.com> wrote:

The "mode" needs to be set to "High Definition Range" for this option to be available.

**Comment by rhys-vdw** [comment]
Ah, cool. I guess maybe if anything it could print a warning when it finds that invalid combo.

Really I feel like Unity devs have created and be maintaining this package.

**Comment by rhys-vdw** [comment]
(It's super useful though, thanks!)

**Comment by AdamFrisby** [comment]
Yeah I'm trying to get a fix for this - I'm running into this myself. The issue is that tonemapping isn't available in LDR (it was before, -ish) --- and I can't port the YRGB curves in HDR (grrr) - I think we need to add a custom PostFX filter which does a YRGB adjustment -after- the colour grading, which allows us to use the curves again.
